cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -708,10 +713,9 @@ func (s *Server) GetInstalledPackageDetail(ctx context.Context, request *corev1.
 	log.Infof("+helm GetInstalledPackageDetail %s", contextMsg)
 
 	namespace := request.GetInstalledPackageRef().GetContext().GetNamespace()
-	cluster := request.GetInstalledPackageRef().GetContext().GetCluster()
 	identifier := request.GetInstalledPackageRef().GetIdentifier()
 
-	actionConfig, err := s.actionConfigGetter(ctx, cluster, namespace)
+	actionConfig, err := s.actionConfigGetter(ctx, request.GetInstalledPackageRef().GetContext())
```

Comment:
Nice! We have `cluster, namespace` in plenty of different places. Good to see how we progressively are evolving to use the Context object (front/back-end)

Reply:
Yeah, it's been nice to gradually simplify calls, not just with context, but also with available/installed package references. Can't do it everywhere in the backend as we're often sharing code with previously working utils, but the closer we get, the less we need those to support both the api and kubeops.